src/components/Tabs.tsx

```diff
@@ -123,15 +130,24 @@ function TabsHorizontal<T extends string = string>(
   props: Omit<TabsProps<T>, 'orientation' | 'opened'>,
 ) {
   const item = useTabsContext();
-
+  const divRef = useRef<HTMLDivElement>(null);
+  const hasScroll = useMemo(
+    () =>
+      divRef.current && divRef.current.scrollWidth > divRef.current.clientWidth,
+    [],
+  );
   return (
-    <div style={{ display: 'flex', flexDirection: 'column' }}>
+    <div
+      style={{ display: 'flex', flexDirection: 'column', overflowX: 'hidden' }}
+    >
       <div
+        ref={divRef}
         style={{
           display: 'flex',
           flexDirection: 'row',
-          height: 38,
+          height: hasScroll ? 60 : undefined,
```

Comment:
This doesn't seem necessary? In the preview, the height is not set and it still seems to work for me:
![image](https://user-images.githubusercontent.com/2352663/195274144-681f1591-5e75-4757-9400-965dc3a0b81b.png)